src/components/ItemForm.jsx

```diff
@@ -0,0 +1,78 @@
+import { useState } from "react";
+import { Button } from "./Button";
+import styles from "./ItemForm.module.css";
+
+const BASE_URL = "http://localhost:9000";
+
+export const ItemForm = ({ setItemsList, itemsList }) => {
+  const [country, setCountry] = useState("");
+  const [city, setCity] = useState("");
+  const [item, setItem] = useState("");
+
+  const postNewItemAndAddItemToState = function (e) {
+    e.preventDefault();
+
+    if (!country || !city || !item) return;
+
+    if (country && city && item) {
+      const newItem = {
+        id: itemsList.length + 1,
+        cityName: city,
+        country: country,
+        item: item,
+      };
+
+      addItem(newItem);
+
+      alert("Item added");
```

Comment:
`alert("Item added");` should also be only triggered if POST is a success

optionally consider putting:
```
  setCountry("");
      setCity("");
      setItem("");
```
into a separate function called clearForm()